tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1646,6 +1657,9 @@ class MklQuantizedConv2DOp
   memory* input_bias_ = nullptr;
   memory* scaled_bias_ = nullptr;
 
+  Tensor scaled_bias_tensor;
```

Comment:
Nit: class member naming. Same for `scaled_bias_buf_`.
```suggestion
  Tensor scaled_bias_tensor_;
```

Reply:
@penpornk Thanks for you comments, I have update my code based on it.